File: Overriding.cpp
#include<iostream>
using namespace std;
class Parent{
    public:
    void print(){
        cout << "i'm the parent class" << endl;
    }  
};
class Child : public Parent{
    public:
    void print(){
        cout << "i'm the child class" << endl;
    }
};
int main(){  
    Child child1;
    child1.print();
}

File: Overriding.java
class Parent {
    void print() {
        System.out.println("i'm the parent");
    }

}

class Child1 extends Parent 
{
    void print() {
        System.out.println("i'm the child");
    }
}

public class Overriding
{
    public static void main(String[] args) {
        Child1 child = new Child1();
        child.print();
    }

}
